packages/workspace/src/browser/i18n.spec.ts

```diff
@@ -0,0 +1,69 @@
+// /*
+//  * Copyright (C) 2017 Ericson and others.
+//  *
+//  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+//  * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//  */
+
+import fs = require('fs');
+import "mocha";
+import * as chai from "chai";
+import { MessageTranslation } from '@theia/core/lib/common/message-translation';
+const Globalize = require("globalize");
+const expect = chai.expect;
+
+Globalize.load(
+    // Core
+    require("cldr-data/supplemental/likelySubtags"),
+    require("cldr-data/supplemental/plurals")
+);
+
+describe('Some menu labels', () => {
+    // Load the json file for each string in different languages
+    loadMessagesfiles();
+    const prefix = "workspace-commands/";
+
+    it('Test Internationalize string in English', () => {
+        MessageTranslation.setLocale("en");
+
+        // Test if the converted string is matching with the proper language
+        expect(MessageTranslation.localize(prefix + "Open", "Error open")).equal("Open...");
```

Comment:
I think we use use the same _localize method ?

Reply:
DONE, Adjusted to use localizeWithPrefix function